docs/user-guide/amor/amor-reduction.ipynb

```diff
@@ -18,6 +18,16 @@
     "## Setup"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')"
```

Comment:
The warnings are raised by `scippnexus` because the transformations in the amor files are not consistent with our concept of nexus (they are lacking `vector` attributes).

In this notebook the warnings are more distracting than relevant, so I thought it best to silence them.

Reply:
But yeah maybe it's better to only silence those warnings, instead of blanket silencing all.